.github/labeler.yml

```diff
@@ -0,0 +1,6 @@
+Documentation:
```

Comment:
For this file, the words above the file path correspond to Labels in the repo found here: https://github.com/giordano/PolynomialRoots.jl/labels. You need to use the existing labels or open an issue requesting the label to be created if it does not exist, like for "Test Suite Modification:". Though, for that one, it might be better to just have a simple "Testing" label.

Reply:
Ok. 
Will do!

Reply:
#17 Request for 2 new labels added. 

One has been pushed. 
Once I'm approved to add those as well, I'll do it.
I was trying to implement the "question" label too but even after learning RegEx to find literal characters, I had no luck. 

Please, can you complete my review with this one label. I assure you I'll add the new labels addressed in the issue which I've opened too.

Thanks!